src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -155,6 +168,20 @@ protected function createPolicy()
         ]);
     }
 
+    /**
+     * Set the table property for the model.
+     *
+     * @param  string  $stub
+     * @param  string  $table
+     * @return string
+     */
+    protected function setTable($stub, $table)
+    {
+        return $table
+            ? str_replace('{{ table }}', 'protected $table = \''.$table.'\';', $stub)
+            : str_replace('{{ table }}', '//', $stub);
```

Comment:
Why not remove the line entirely in the `else` case?

Reply:
I thought it would be needlessly complex to do that. What do you think?